eng/Versions.props

```diff
@@ -100,6 +100,17 @@
       </PropertyGroup>
     </Otherwise>
   </Choose>
+  <!--
+    Patched package versions used by src\RoslynAnalyzers\Tools\Metrics*
+
+    Metrics targets the same older Microsoft.CodeAnalysis version as the RoslynAnalyzers. Analyzers pick up patched versions
+    of Roslyn's transtive dependencies from the compiler that is hosting them. Since Metrics is a standalone tool and not an
+    Analyzer, it needs to directly reference patched versions of some transitive dependencies.
```

Comment:
Is this comment correct? Because in the case of these two dependencies (System.Text.Json and Microsoft.Build) there is no _compiler_ host verson it'd be using, right? So it will actually use that version?